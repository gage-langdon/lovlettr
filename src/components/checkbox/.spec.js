import React from 'react';
import { shallow } from 'enzyme';

//component
import Checkbox from './index';

describe('Checkbox component', () => {
  it('Will render with hidden passed in, will be hidden', () => {
    const wrap = shallow(<Checkbox hidden={true} />);
    expect(wrap).toMatchSnapshot();
  });
});
