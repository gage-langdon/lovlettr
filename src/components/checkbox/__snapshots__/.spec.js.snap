// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox component Will render with hidden passed in, will be hidden 1`] = `
<div
  onClick={[Function]}
>
  <test-file-stub
    className="tick"
    hidden={true}
  />
  <div
    className="tick-container"
  />
</div>
`;
